app/Http/Controllers/Auth/AuthController.php

```diff
@@ -126,7 +127,9 @@ public function requestOtp(Request $request)
             'mobile' => 'required|numeric',
         ]);
         $newNumber = ltrim($request->newnumber, '0');
-        User::where('id', $request->id)->update(['mobile' => $newNumber]);
+        $newCode = ltrim($request->code, '0');
```

Comment:
This code is repeating, you can create a protected method in parent class and call it in both parent and child class

Reply:
No need of ltrim() function for mobile, so removed, just mentioned as $request->code.

Reply:
Not ltrim, i am talking about the whole code block